src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.html

```diff
@@ -212,6 +212,14 @@ <h4 *ngIf="isImport" id="jhi-programming-exercise-heading-import" jhiTranslate="
                         </label>
                     </div>
                 </div>
+                <div class="form-group-narrow" *ngIf="!isInProduction()">
```

Comment:
please add a HTML comment
<!-- NOTE: this is only to be used in dev environments for testing purposes -->

Reply:
i added comments to all modified HTML files